etc/default/default_config.yaml

```diff
@@ -2,14 +2,14 @@
  # Logging
 vectoria_logs_dir: ~/vectoria_logs
 log_level: DEBUG
-langchain_tracking: null
+langchain_tracking: true
```

Comment:
Do you recommend setting this to true by default? If so let me know so I can update the ansible template as well